flytekit/core/context_manager.py

```diff
@@ -600,6 +602,11 @@ def current_context() -> Optional[FlyteContext]:
         """
         return FlyteContextManager.current_context()
 
+    def display_deck(self):
```

Comment:
lets just make it - get_deck, instead of display.
I think we can ask users to use 
```
with ... as ctx:
      t1()
      display(ctx.deck())
```